gef-ui/templates/partials/confirm-cover-start-date.njk

```diff
@@ -0,0 +1,53 @@
+{% extends "index.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% block pageTitle %}
+    Confirm cover start date for issued facilities
+{% endblock %}
+
+{% block content %}
+
+    {% if errors %}
+        {{
+            govukErrorSummary({
+            titleText: "There is a problem",
+            errorList: errors.errorSummary,
+            attributes: {
+                'data-cy': 'error-summary'
+                }
+            })
+        }}
+    {% endif %}
+
+    {% set hide_back = 1 %}
+    {% include "includes/application-preview/summary.njk" %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full-from-desktop">
+            {{ govukBackLink({
+                text: "Back",
+                href: "/gef/application-details/" + applicationId,
+                attributes: {
+                'data-cy': 'back-link'
+                }
+            }) }}
+        </div>
+    </div>
+
+        <header>
+            <h1 data-cy="confirm-cover-start-date-heading">Confirm cover start date for issued facilities</h1>
+        </header>
+
+    {% if isUkefReviewAvailable %}
+        <div class="govuk-grid-row govuk-body">
+            <div class="govuk-grid-column-two-thirds">
+                <form method="POST" autocomplete="off">
```

Comment:
do we need this empty form here?

Reply:
(or is it for the next ticket?

Reply:
This part of my ticket (AC8), however part 1 wil populate the fileds.